lisa/operating_system.py

```diff
@@ -167,7 +263,16 @@ def install_packages(
         assert isinstance(packages, list), f"actual:{type(packages)}"
         for item in packages:
             if isinstance(item, str):
-                package_names.append(item)
+                # if item is a URL, download the package first
+                # and then append the downloaded file to package_names.
```

Comment:
As we discussed, please make a separated method for url installation. So sometime it doesn't need to call wget.

Reply:
> As we discussed, please make a separated method for url installation. So sometime it doesn't need to call wget.

Done